apps/src/templates/projects/ProjectsList.story.jsx

```diff
@@ -27,41 +32,26 @@ const STUB_PROJECTS_DATA = [
   }
 ];
 
-export default storybook => {
-  return storybook.storiesOf('Projects/ProjectsList', module).addStoryTable([
-    {
-      name: 'basic projects list without thumbnails column',
-      description: `This is a simple projects list with stub data.`,
-      story: () => (
-        <ProjectsList
-          projectsData={STUB_PROJECTS_DATA}
-          studioUrlPrefix={'https://studio.code.org'}
-          showProjectThumbnails={false}
-        />
-      )
-    },
-    {
-      name: 'basic projects list with thumbnails column',
-      description: `This is a simple projects list with stub data.`,
-      story: () => (
-        <ProjectsList
-          projectsData={STUB_PROJECTS_DATA}
-          studioUrlPrefix={'https://studio.code.org'}
-          showProjectThumbnails={true}
-        />
-      )
-    },
-    {
-      name: 'basic localized projects list',
-      description: `This is a simple projects list with stub data, rendered in a non-English locale`,
-      story: () => (
-        <ProjectsList
-          locale={'es-MX'}
-          projectsData={STUB_PROJECTS_DATA}
-          studioUrlPrefix={'https://studio.code.org'}
-          showProjectThumbnails={true}
-        />
-      )
-    }
-  ]);
+const DEFAULT_ARGS = {
```

Comment:
I'm curious if there is a benefit to having these DEFAULT_ARGS or if we can just put them in the template itself, like
```
<ProjectsList
  projectsData: STUB_PROJECTS_DATA,
  studioUrlPrefix: 'https://studio.code.org',
  showProjectThumbnails: true
  {...args}
/>
```